components/Navbar/Navbar.js

```diff
@@ -1,14 +1,16 @@
 import Link from "next/link";
 
 import { FaBars } from "react-icons/fa";
-import { MobileIcon, Nav, StyledLink } from "./NavbarElements";
+import { MobileIcon, Nav, NavIcon, StyledLink } from "./NavbarElements";
 
 const Navbar = ({ toggle }) => {
   return (
     <Nav>
       <div>
         <Link href="/" passHref>
-          <StyledLink>Logo</StyledLink>
+          <StyledLink>
+            <NavIcon src="/logo.svg" />
```

Comment:
instead of passing `.svg` to `img` element create a react funtion that return this svg. Example here: https://www.pinkdroids.com/blog/svg-react-styled-components/